gopherage/pkg/cov/junit/calculation/coveragelist.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package calculation
+
+import (
+	"path"
+	"sort"
+	"strings"
+)
+
+// CoverageList is a collection and summary over multiple file Coverage objects
+type CoverageList struct {
+	*coverage
+	Group             []coverage
+	CoverageThreshold int
```

Comment:
It feels a bit strange that `CoverageThreshold` is a property of the coverage information. Would it make more sense for this to be managed elsewhere by the code that uses the thresholding (e.g. in `junit`).